IBMiMake

```diff
@@ -749,6 +759,10 @@ $(DEPDIR)/%.rebuild:
 #make_pre:
 #	mkdir -p $(LOGPATH)
 
+.PHONY: clean
+clean:
+	rm -rf ./.deps ./evfevent ./logs
+	rm -rf $(OBJPATH)/*
```

Comment:
pretty dangerous to delete the contents of the object library.  I would comment that out